unlock-app/src/components/content/SettingsContent.tsx

```diff
@@ -73,7 +82,7 @@ export class SettingsContent extends React.Component<
         <AccountInfo />
         <ChangePassword />
         {cards.length > 0 && <PaymentMethods cards={cards} />}
-        {!cards.length && <PaymentDetails stripe={stripe} />}
+        {stripe && !cards.length && <PaymentDetails stripe={stripe} />}
```

Comment:
This is the real meat of the change. Something about the transition from null to Stripe makes `react-stripe-elements` go wild and break the app.